src/output/output.cpp

```diff
@@ -51,7 +51,7 @@ static wl_output_transform get_transform_from_string(std::string transform)
     else if (transform == "180")
         return WL_OUTPUT_TRANSFORM_180;
     else if (transform == "270")
-        return WL_OUTPUT_TRANSFORM_180;
+        return WL_OUTPUT_TRANSFORM_270;
```

Comment:
why is that in this PR? maybe I forgot to push something

Reply:
Idk. I think it merged in some commit at some point...

Reply:
yeah if you rebase it should automatically drop this since it isn't part of this PR